src/util/process.cpp

```diff
@@ -32,7 +32,7 @@ void process::run() {
         buffer<char*> pargs;
         for (auto arg : m_args) {
             std::cout << arg << std::endl;
```

Comment:
The whole loop body can be written as just `pargs.push_back(arg.c_str())`.  The pointer returned by `c_str` is valid as long as we own the string `arg` and do not pass it as a mutable reference somewhere.